docs/governance/temporary-voting.md

```diff
@@ -0,0 +1,161 @@
+  through the lifetime of a proposal as it moves through this process.
+
+At this time, the community is invited to:
+
+* Recommend additional goals, or discuss nuances of the stated goals the working
+  group has already shared
+* Recommend implementation details
+
+The working group will consider these recommendations in good faith, but may
+choose not to adopt them.
+
+### Stage 2: Proposal Review Period
+
+This stage lasts until the working group feels major concerns have been
+addressed and the proposal is ready for a vote. However, **at least 72 hours**
+must elapse between the proposal being published and the vote starting, to allow
+community members around the globe to read and comment. Typically, this stage
+should last at least one week.
+
+At this time, the working group should:
+
+* Share the full text of the proposal as a GitHub pull request (PR)
+* Link this GitHub PR to the existing Discord notification thread and GitHub
+  discussion
+* Explain how the proposal meets the goals stated in Stage 1, either within the
+  proposal itself or in commentary next to the proposal
+
+At this time, the community is invited to:
+
+* Share constructive recommendations in GitHub to modify the text of the
+  proposal, or discuss nuances of the proposal's wording
+* Share showstopper concerns in GitHub about the proposal, including details
+  about how and why the concern is especially dire
+
+Think of this like a code review; the goal of this stage is to build a proposal
+that is representative of the community's will. Keep recommendations actionable
+and constructive: "This clause discourages X; if we phrase it like "foo bar baz"
+it could be less exclusive" is much more productive than "It's obvious that the
+governance working group doesn't want X!"
+
+At the discretion of the working group, but based on the outcome of the
+discussion, the proposal will go to a vote **or** the proposal will be dropped.
+
+### Stage 3: Proposal Voting Period
+
+When the working group feels that major concerns have been addressed and is
+happy with the text of the proposal, the working group will open voting on the
+proposal.
+
+* Voting occurs on GitHub using the poll feature and is advertised heavily on
+  Discord during the voting period.
+  * If community members want to vote but aren't able to use GitHub, they can
+    message nasamuffin@ (on Discord, or nasamuffin at google dot com) with their
+    vote to have it manually included. Only one working group member is listed
+    in order to avoid accidental double-counting.
+  * When voting against, community members should comment on the post explaining
+    why and describe what change would be required for them to abstain or vote
+    in favor.
+  * Generally, assume that the votes may be publicly visible or may be made
+    publicly visible at a later time.
+* Voting is open for 1 week, after which point the GitHub poll will be locked.
```

Comment:
Minor note: when possible and for larger proposals, it might be nice to have the option of the voting lasting longer than a week, e.g. to include 2 weekends, or even closer to 2 weeks.

So, I'd adjust these rules to something like:

- Voting is open for a minimum of 1 week, longer when appropriate (for less urgent, larger and/or more complex proposals).
- The deadline is announced at the beginning of the voting period.

That said, I'm OK with the proposal as is if anyone prefers that.

Reply:
Yeah, that seems fine to me. I'm going to rephrase it to put an explicit mention of capturing two weekends as a reason to extend the voting deadline, will send a new version sometime today.